### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by Carroway256**

### CODE REVIEW

**Comment by Carroway256** [comment]
mis-1 basic backend

File: services/passport.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+const passport = require("passport");
+const GoogleStrategy = require("passport-google-oauth20").Strategy;
```
Comment (cryptowizard) [inline services/passport.js:3]: Why is anything from Google in here?

**Comment by Carroway256** [comment]
😂😂 it was supposed to be .gitignored
It was part of practice exercise

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+To install all dependencies run npm install
+to start server run npm start
+server will start at localhost:5000
+to search for querry use URL http://localhost:5000/search/QUERRY_HERER
```
Comment (cryptowizard) [inline readme.md:4]: Please make it so its a REST GET request thats accepts JSON as a body argument. 
It should not require any autorization for now.
You should be able to use POSTMAN to test it.

**Review by cryptowizard — CHANGES_REQUESTED** [review]

**Pull request merged**